src/js/angular/ttyg/controllers/agent-settings-modal.controller.js

```diff
@@ -262,8 +268,19 @@ function AgentSettingsModalController(
 
     $scope.checkIfFTSEnabled = () => {
         if (!$scope.agentFormModel.repositoryId) {
+            $scope.agentSettingsForm.$setValidity('FTSDisabled', false);
```

Comment:
You must add this in the template for the FTS method as well `ng-class="{'has-error': agentSettingsForm.$error.FTSDisabled}"`

Reply:
Now I see that this is missing also for the similarity search panel. Add it there too please